prerequisites.rst

```diff
@@ -1,35 +1,250 @@
+interface (SATA, ISE, SCSI, USB, SERIAL) but information and management will
+depend on the disk drives supported by the Debian operating system.
+
+Modern hard disks drives have firmware inside that reports several attributes, 
+Any hard disks drives is supported but |omv| only supports SMART for those
+disks that are connected to an HBA in pass-through mode.
+
+The system manages two types of storage unit classification:
+
+System drive storage
+  The Storage Disk Drive(s) used to put the system program files (by partitions), 
+  **this system drive (that is in fact a couple or more partitions) 
+  cannot officially be used for shared resources or as data drive**.
+  What supported brands and sizes are covered in the software requirements section below.
+  The recommendation for the System drive(s) hardware could be older spinning
+  Hard Disk Drives (HDD), Disk-on-Module [3]_, CompactFlash [4]_ or thumb drives (USB),
+  the Solid State Disks (SDD [2]_) must be managed.
+
+Data drive storage
+  The storage disk drive(s) where the data for the defined or used shares resources
+  wil be stored. **Cannot be the same of the system drive(s) and sizes will
+  depends of the usage of the resources**. Shared resources are managed over
+  partitions, which is discussed in the software requirements topic below.
+  The recommendation for the Data drives must be Solid State Disks (SSD [2]_ disks)
+  for best performance or spinning Hard Disk Drives (HDD).
+
+The memory (RAM)
+^^^^^^^^^^^^^^^^
+
+Enough RAM is vital to maintaining peak performance. There are several combinations
+of installation sizes but the **recommendation is at least 4Gib for novice administrators**.
+
+For best practice you should have 8GiB of RAM for basic operations in default installation.
+and no matter the size, RAM must be configured in Dual Channel mode [8]_ 
+inclusivelly if there are little amount of, it will improve performance.
+
+Unless other NAS systems the |omv| system can run in at least 1Gib of RAM, but
+of course will need expertise.
+
+The communication card (NIC)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A NAS system as means is a network oriented system, of course |omv| is the
+most flexible NAS system in the world and allows multiple ways of communication, 
+this is becouse |omv| offers several modes of shared resources like RSYNC that 
+do not depends on networking. But there are few key points:
+
+- The NAS write speed to storage is limited to the speed of your NAS computer NIC card
+- The NAS write speed to storage is limited to the speed of your network
+- Speed of writing to the storage depends on ratio of the number of users accessing it.
+
+Full-duplex NICs are recommended, half-duplex NICs typically offer 10/100 Mbit/s
+and storage drives far exceed those speeds but are of course supported by |omv|.
+
+Any supported NIC vendor is determined by the support of the operating system
+where |omv| system resides and runs, which is Debian GNU/Linux.
+
```

Comment:
You should add a note that older hardware is better supported by Debian out-of-the box. New hardware which is new to the market might not be supported by the default Linux kernel and has administrative hurdles because it requires manual driver integration.

Reply:
fix with commit c441ec7d9649da3fb60e9f23635c041b62f2a54c